### ISSUE TITLE
0.7.1 breaks import in Nuxt projects

### ISSUE THREAD

**Issue by marc1404** [monterail/vuelidate#301]
If I upgrade vuelidate from `v0.7.0` to `v0.7.1` it breaks the vuelidate import in Nuxt based projects:
```javascript
{ .../node_modules/vuelidate/lib/validators/index.js:1
(function (exports, require, module, __filename, __dirname) { import alpha from './alpha';
                                                                     ^^^^^

SyntaxError: Unexpected identifier
    at new Script (vm.js:74:7)
    at createScript (vm.js:246:10)
    at Object.runInThisContext (vm.js:298:10)
    at Module._compile (internal/modules/cjs/loader.js:646:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:689:10)
    at Module.load (internal/modules/cjs/loader.js:589:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:528:12)
    at Function.Module._load (internal/modules/cjs/loader.js:520:3)
    at Module.require (internal/modules/cjs/loader.js:626:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at r (.../node_modules/vue-server-renderer/build.js:8330:16)
    at Object.<anonymous> (server-bundle.js:1507:18)
    at __webpack_require__ (webpack:/webpack/bootstrap c2794e5cacb35c2b7440:25:0)
    at Object.<anonymous> (src/form/LocalizedField.vue?1327:1:0)
    at __webpack_require__ (webpack:/webpack/bootstrap c2794e5cacb35c2b7440:25:0)
    at Object.module.exports.render (src/form/LocalizedField.vue:1:0) statusCode: 500, name: 'SyntaxError' }
```
The previous version used CommonJS in `lib/validators.index.js` whereas `v0.7.1` introduces ES modules syntax with `import`.
Is this intended?

**Comment by sh** [comment]
I've had the same problem, coming from `v0.6.2` to `v0.7.1`

I've added a fix via #303, would love to see it published soon :)

**Comment by marc1404** [comment]
Awesome @sh ! :)
